client/tokenized-payment-request/blocks/use-initialization.js

```diff
@@ -70,8 +75,9 @@ export const useInitialization = ( {
 		paymentRequest,
 		billing?.cartTotal?.value,
 		isFinished,
-		shippingData?.needsShipping,
-		billing?.cartTotalItems,
+		shippingData.needsShipping,
+		billing.cartTotalItems,
```

Comment:
```suggestion
		shippingData?.needsShipping,
		billing?.cartTotalItems,
```
Any reason why we removed the conditional safety checks here? A few lines above we check for `billing?.cartTotal?.value`, which suggests that `billing` _might_ be undefined. It doesn't appear that we are adding the `billing` or `shippingData` props ourselves AFAICT, so can we be certain we can rely on them? 🤔

Reply:
No reason to remove the checks here. I often use eslint to automatically fix dependency arrays and it looks like it might've stripped these out for some reason. Good to know! - [51ffe7c](https://github.com/Automattic/woocommerce-payments/pull/9150/commits/51ffe7c0147196b7a761021e821d8091b5652e94)